app/services/coronavirus_pages/content_builder.rb

```diff
@@ -97,4 +98,22 @@ def add_live_stream(data)
       data["live_stream"] = github_live_stream_data.merge(persisted_live_stream_data)
     end
   end
+
+  def hidden_search_terms
+    sections = sub_sections_data.map do |section|
+      [section[:title], subsections(section[:sub_sections])]
+    end
+
+    sections.flatten.compact
+  end
+
+  def subsections(items)
```

Comment:
I wonder if this method name is a little broad, since it only relates to `hidden_search_terms`. I notice there is already a [`sub_sections_data` method](https://github.com/alphagov/collections-publisher/blob/1e06a43a6156c11cd0bfcaa5b0118da49d667c68/app/services/coronavirus_pages/content_builder.rb#L80) that feels quite similar in naming to this.